app/main/views/services.py

```diff
@@ -157,6 +157,18 @@ def update_section(service_id, section):
                 "user",
                 "supplier app")
         except HTTPError as e:
+            errors_map = {}
+            for error in e.response.json()['error'].keys():
+                id = existing_service_content.get_question(error)['id']
```

Comment:
How do you provoke a `_form` error here? Do you have to hack the posted data or is there input you can make on a form that will break it?